metrics-dropwizard/src/main/scala/com/mesosphere/usi/metrics/dropwizard/DropwizardMetrics.scala

```diff
@@ -18,11 +18,16 @@ class DropwizardMetrics(metricsSettings: MetricsSettings, registry: MetricRegist
 
   private val namePrefix = metricsSettings.namePrefix
   private val histrogramSettings = metricsSettings.historgramSettings
-  private val histogramReservoirHighestTrackableValue = histrogramSettings.reservoirHighestTrackableValue
-  private val histogramReservoirSignificantDigits = histrogramSettings.reservoirSignificantDigits
-  private val histogramReservoirResetPeriodically = histrogramSettings.reservoirResetPeriodically
-  private val histogramReservoirResettingInterval = Duration.ofNanos(histrogramSettings.reservoirResettingInterval.toNanos)
-  private val histogramReservoirResettingChunks = histrogramSettings.reservoirResettingChunks
+  private val histogramReservoirHighestTrackableValue =
+    histrogramSettings.reservoirHighestTrackableValue
+  private val histogramReservoirSignificantDigits =
+    histrogramSettings.reservoirSignificantDigits
+  private val histogramReservoirResetPeriodically =
+    histrogramSettings.reservoirResetPeriodically
+  private val histogramReservoirResettingInterval =
+    Duration.ofNanos(histrogramSettings.reservoirResettingInterval.toNanos)
+  private val histogramReservoirResettingChunks =
+    histrogramSettings.reservoirResettingChunks
```

Comment:
ugh

Reply:
That's just line length and the same as on the left. We could wrap it with `//format:OFF`, though.